lib/gui/components/drive-selector/templates/drive-selector-modal.tpl.html

```diff
@@ -51,6 +51,13 @@ <h4 class="list-group-item-heading">{{ drive.description }} - {{ drive.size | gi
           ng-show="modal.constraints.isDriveValid(drive, modal.state.getImage())"
           ng-disabled="!modal.state.isCurrentDrive(drive.device)"></span>
     </li>
+    <li class="list-group-item"
+      ng-show="modal.drives.getDrives().length === 0">
+      <div>
+        <b>Connect a drive!</b>
+        <div>No removable drive detected.</div>
```

Comment:
I guess it could be either, given that the title is specifically asking "Connect a drive".
But in the general case, you'd use 0 drives, 1 drive, 2 drives, 3 drives, ...